tests/testnet-calls.py

```diff
@@ -0,0 +1,343 @@
+from web3 import Web3
+from web3.exceptions import ContractCustomError, ContractLogicError
+import json
+from eth_account.messages import encode_defunct
+import time
+import sys
+import os
+import random
+from eip712.messages import EIP712Message
+from helpers import (
+    TEST_NET_VARS,
+    check_subgraph_signer,
+    check_deployed_subgraph_transaction
+)
+test_net_type = sys.argv[1]
+testnet = "eth-goerli"
+if(test_net_type != 'eth'):
+    testnet = "aribitrum-goerli"
```

Comment:
It would be simpler and more reliable to just get it directly.
```suggestion
testnet = sys.argv[1]
# If you really want to check early:
assert testnet in ("eth-goerli", "aribitrum-goerli")
```

Reply:
Yeah was thinking so that it be easier whenever the command is ran, but its better to just write down in the readMe that the full testnet name is expected